index.js

```diff
@@ -5,46 +5,48 @@ var through = require('through2');
 var minimatch = require("minimatch");
 var _ = require('lodash');
 
-function instrument(options) {
-  var Instrumenter = require(resolve.sync(options.instrumenter, {basedir: __dirname}));
+function filterFiles(options, files) {
   var excludePattern = options.exclude ? [].concat(options.exclude) : [''];
```

Comment:
oh I didn't notice that line. Sorry about that. That makes sense. Lets merge then so I publish the newest version.
Are you trying to fix the test right now or we do it after publishing? I'm not in a hurry.